starter-code/scripts/views/articleView.js

```diff
@@ -35,6 +39,7 @@
   };
 
   // COMMENT: What does this method do?  What is it's execution path?
+  //This method articleView object is defining the function handleFilters which takes in no parameters and queries the filters ID and runs the jQuery method one with an event of change, the data parameter select, and a handler which is an anonymous function that defines resource as a variable equaling to this.id.replace('-filter', '') which replaces filter with an empty string. Then queries the parent element of this which is the parent element of #filters and sets the value of it's siblings children equal to an empty string. So then we are calling the page function and setting the URL to resource/this.val and replacing any whitespace with a plus sign.
   articleView.handleFilters = function() {
     $('#filters').one('change', 'select', function() {
```

Comment:
The jQuery.one method is adding an event handler, similar to doing a jQuery.on - the difference being that it should run only a single time, then unbind the handler from the element. 'select' is not a data parameter, it's an optional selector for descendants of the element we queried (think how we set up delegated event handlers for children of elements). Keep in mind that #filters has two select element children - select#author-filter and select#category filter.

So the first time the value of one of these select elements is changed, we're grabbing the ID ("author-filter" or "category-filter"), and replacing "-filter" with "", hence getting either "category" or "author".

As a result, $(this) will refer to whichever select element we clicked on - so $(this).parent() will take us back to the <li> element that held the select box. Its sibling is the other <li> element, and the child of that sibling is the other select element. So what we're doing is saying, "Whenever a select element is changed, grab the value of what it got changed to, then set the other select element back to having no value" - e.g., if we select an author, change our category filter back to "select a category."